R/fetcher.R

```diff
@@ -485,16 +488,17 @@ fetch_emendas_senado <- function(bill_id) {
 #' @description Retorna dataframe com os dados da tramitação de uma proposição no Congresso
 #' @param id ID de uma proposição na sua respectiva casa
 #' @param casa Casa onde a proposição está tramitando
+#' @param normalized whether or not the output dataframe should be normalized (have the same format and column names for every house)
 #' @return Dataframe com os dados da tramitação de uma proposição no Congresso
 #' @examples
 #' fetch_tramitacao(91341,'senado')
 #' @export
-fetch_tramitacao <- function(id, casa, normalize=FALSE) {
+fetch_tramitacao <- function(id, casa, normalized=FALSE) {
```

Comment:
agoradigital::fetch_tramitacao(91341,'senado', TRUE)
Error in mutate_impl(.data, dots) : 
  Evaluation error: object 'numero_ordem_tramitacao' not found.
Quando eu rodo só agoradigital::fetch_tramitacao(91341,'senado') pega